client/apps/articles_list/routes.coffee

```diff
@@ -6,17 +6,20 @@ Transport = require('lokka-transport-http').Transport
 
 @articles_list = (req, res, next) ->
   channel_id = req.user?.get('current_channel').id
-  publishedQuery = query "published: true, channel_id: \"#{channel_id}\""
+
+  if req._parsedUrl.query?.includes 'published=false' then published = false else published = true
```

Comment:
i think you can use `req.query` here for the express-y way. i believe it's the safer option after doing a quick search for `_parsedUrl`. It seems like it's added as a caching optimization, so _might_ not be as reliable? i'm not sure..